src/package/cppwinrt/vsix/ProjectTemplates/VC/Windows Desktop/ConsoleApplication/pch.h

```diff
@@ -1,2 +1,4 @@
 ﻿#pragma once
+#include <winrt/coroutine.h>
 #include <winrt/Windows.Foundation.h>
+#include <winrt/Windows.Foundation.Collections.h>
```

Comment:
Namespace headers always include their parent namespace headers.

Reply:
yes, but I like the explicit calling out so users can selectively remove one at a time